tools/test.py

```diff
@@ -1562,12 +1562,12 @@ def ArgsToTestPaths(test_root, args, suites):
   return paths
 
 
-def get_env_type(vm, options_type):
+def get_env_type(vm, options_type, context):
   if options_type is not None:
     env_type = options_type
   else:
-    if "fips" in subprocess.check_output([vm, "-p",
-                                          "process.versions.openssl"]):
+    ssl_ver = Execute([vm, "-p", "process.versions.openssl"], context).stdout
+    if "fips" in ssl_ver:
```

Comment:
Since you're here, can you use single quotes?

Reply:
ack